# cepheus:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
  # Effort Controllers ---------------------------------------
  # thrusterx_effort_controller:
  #   type: effort_controllers/JointEffortController
  #   joint: thrusterx_joint
  #   pid: {p: 10.0, i: 0.1, d: 1.0}
  # thrustery_effort_controller:
  #   type: effort_controllers/JointEffortController
  #   joint: thrustery_joint
  #   pid: {p: 10.0, i: 0.1, d: 1.0}
  reaction_wheel_effort_controller:
    type: effort_controllers/JointEffortController
    joint: reaction_wheel_joint
    pid: {p: 10, i: 0.1, d: 1.0}
  left_shoulder_effort_controller:
    type: effort_controllers/JointEffortController
    joint: left_shoulder_joint
    pid: {p: 10.0, i: 0.1, d: 1.0}
    # pid: {p: 1.0, i: 0.1, d: 1.0}
  left_elbow_effort_controller:
    type: effort_controllers/JointEffortController
    joint: left_elbow_joint
    pid: {p: 10.0, i: 0.1, d: 1.0}
  left_wrist_effort_controller:
    type: effort_controllers/JointEffortController
    joint: left_wrist_joint
    pid: {p: 10.0, i: 0.1, d: 1.0}
    # pid: {p: 1.0, i: 0.1, d: 1.0}
